### DISCUSSION TITLE
Suddenly not finishing forms anymore

### DISCUSSION THREAD

**Discussion by lupusworax** [arkypita/LaserGRBL#1882] [Q&A]
I was working with my laser cutting paper with lots of cutouts and forms inside. Everything was working perfectly, but from one moment to another the laser now does not finish cutting the smaller forms.  I have it set to constant power, and it worked perfectly until it changed. When cutting a circle for example it does not finish the form, stops 4-5 mm before and changes position to the next cut. I tried files it was working perfectly before and now its the same thing and I just cannot find whats causing the issue.  I reinstalled laserGRBL tuned, rebootet windows, switched the lasercutter on off ( Diode laser)  but no success.  Any suggestions what could suddenly cause my issue?  Thanks in advance.

**Comment by Gelenkbusfahrer** [comment]
Hi,
I'm pushing this up since i have a similar problem when cutting paper with the Atomstack S10pro. From my observations a white paper causes more problems than a darker one. Black paper will be cut without the issue. I also observed that with less laser power the left out parts of the cut are getting bigger. So 10% will produce bigger dropouts than 30%. the laser seems to start delayed after it has been moved to a new position. This is also happening pretty randomly. approx 5-10 of 100 test circles on one paper sheet are incomplete. This sounds weird, no?
Thanks for any ideas!

**Reply by arkypita** [reply]
There is nothing strange in this: white reflects laser light, black absorbs it.
Review the basic notions of physics (we see an object white because white reflects all light wavelengths, we see an object black because it absorb all light wavelengths, we see an object red because it absorbs all wavelengths except the red ones) and also review the common experience: a white car and a black car left in the sun reach different temperatures.

Engraving transparent, white, or blue materials with blue laser diode is always hard.

**Comment by Gelenkbusfahrer** [comment]
thanks! yes, cutting white paper with more power helps but introduce other problems like overburned cuts.
